### DISCUSSION TITLE
Restart after Quota Limit

### DISCUSSION THREAD

**Discussion by StackShipping** [cline/cline#1032] [Feature Requests]
Hi,

I am using cline with gemini 2.0 flash experimental so its free but I keep running into this:
[GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1beta/models/gemini-2.0-flash-exp:streamGenerateContent?alt=sse: [429 Too Many Requests] Resource has been exhausted (e.g. check quota).

Then I need to wait a minute and click again. 

Can you guys add a feature to do this automatically? That would save a bunch of time!

Thanks a lot.

Great progress guys!

**Comment by skele6011** [comment]
Having the same problem here but with 1.5 Flash, for me it's slowly starting to happen more frequently. Like, I could put in two texts, and in the third one it breaks.

**Comment by sunlast** [comment]
Have same issue with gemini flash 2. [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1beta/models/gemini-2.0-flash-001:streamGenerateContent?alt=sse: [429 Too Many Requests]. I check my quote on the google webpage and it is nowhere near max.
